### ISSUE TITLE
batch geocoding

### ISSUE THREAD

**Issue by fauntle** [mapbox/mapbox.js#708]
When I use semicolons in a mapbox geocoder query, I never get a successful response. I can easily do this using the API proper, so why can't I batch geocode with mapbox.js?

``` js
var geocoder = L.mapbox.geocoder( MAP_ID );
geocode.query( 'San Francisco;Oakland', function( err, data ){
    // err always returns `true`
    console.log( err, data );
});
```

**Comment by tmcw** [comment]
Indeed, there's a bug here where mapbox.js isn't correctly parsing the response from multi-geocodes.

**Comment by tmcw** [comment]
Fixed in 9a4c54cd715ac20eb34b57dd8373319f1d01164c
